docs/content/Getting-Started/Core/04-Query-Cube-API.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Query Cube API
+permalink: /getting-started/core/query-cube-api
+category: Getting Started
+subCategory: Core
+menuOrder: 6
+isDisableFeedbackBlock: true
+---
+
+## Developer Playground
+
+The [Developer Playground](/dev-tools/dev-playground) is a web-based tool which
+allows for model generation and data exploration.
+
+Select the measures and dimensions you want to query, and then run the query.
+Let's do this for the `Orders` cube you generated in the previous step.
+
+Click **+ Measure** to display the available measures and add `Orders.count`,
+then click **+ Dimension** for available dimensions and add `Orders.status`.
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Deployment Developer Playground Screen with selected measures and dimensions"
+    src="https://ucarecdn.com/0c5af31d-9b33-4e4c-9192-966d3a315d02/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+Then click the **Run** button to execute the query and see the results:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Deployment Developer Playground Screen showing query results"
+    src="https://ucarecdn.com/81cccc1f-9153-4917-920d-feea0de6591b/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+## GraphQL API
+
+Cube supports using GraphQL to query cubes via the `/graphql` endpoint:
+
+```bash
+curl \
+  --data-raw '{"query":"{cube{orders{count}}}"}' \
+  https://localhost:4000/cubejs-api/graphql
+```
+
+## REST API
+
+Cube supports using GraphQL to query cubes via the `/load` endpoint:
+
+```bash
+curl \
+  --data-urlencode 'query={"measures":["Orders.count"]}' \
+  https://fierce-boar.aws-eu-central-1.cubecloudapp.dev/dev-mode/dev-hassan-45a93e7e/cubejs-api/v1/load
+```
+
+## SQL API
+
+To enable the SQL API, we first need to add a few new environment variables to
```

Comment:
These variables could/should be added in the default `docker-compose.yml` file, so there's no need to have this part here.

Reply:
The default setup with Docker Compose automatically creates a `.env` file with all the database credentials, that's why we're adding it there